tests/test_api.py

```diff
@@ -223,7 +224,8 @@ def setUp(self):
         (False, False, False, False),
     )
     @ddt.unpack
-    @patch('learning_assistant.api.learning_assistant_available')
+    @patch('learning_assistant.api.learning_assistant_available_flag')
+    @override_settings(LEARNING_ASSISTANT_AVAILABLE=False)
```

Comment:
Do these tests fail if `LEARNING_ASSISTANT_AVAILABLE=True`? Or are you just trying to ensure that these tests are testing the waffle flag logic and not the Django setting?

Reply:
I was trying to limit the scope of this test, and not introduce more booleans to test the logic of the django setting. Although now I'm realizing because I added a test for the api function `learning_assistant_available`, I could mock that here, and avoid overriding the setting.